dateutil/rrule.py

```diff
@@ -798,6 +826,27 @@ def _iter(self):
         total = 0
         count = self._count
         while True:
+            leap = False
+
+            # SKIP rule
+            if day > 28 and self._skip is not None and self._skip != 'OMIT':
+                daysinmonth = calendar.monthrange(year, month)[1]
+                if day > daysinmonth:
+                    if self._skip == 'BACKWARD':
+                        day = daysinmonth
+                        leap = True
```

Comment:
Why is this called `leap`? Shouldn't it be called `skipped`? Or `shifted` or something?